packages/apollo-codegen-scala/src/codeGeneration.ts

```diff
@@ -53,6 +53,7 @@ import { createLexer } from "graphql/language";
 export function generateSource(context: LegacyCompilerContext) {
   const generator = new CodeGenerator(context);
 
+  generator.printOnNewline("// @generated");
   generator.printOnNewline(
     "//  This file was automatically generated and should not be edited."
```

Comment:
alternatively, it can be added to the same line (similar to jsdoc tags format):
```
//  @generated This file was automatically generated and should not be edited.
```

Reply:
I think it is better to explicitly mark this line, to avoid removing when someone decide to change the message about file.